TypeCobol/Compiler/Parser/CodeElementBuilder/CobolWordsBuilder.cs

```diff
@@ -411,6 +411,14 @@ internal SymbolDefinitionOrReference CreateSymbolDefinitionOrReference(CodeEleme
             return symbolDefinitionOrReference;
         }
 
+        internal SymbolDefinitionOrReference CreateSymbolDefinitionOrReference(ITerminalNode node, SymbolType symbolType)
```

Comment:
Yes, this new method can be used to factorize code with the two existing ones.

Reply:
An attempt of factorization...